Spectral/style.css

```diff
@@ -0,0 +1,473 @@
+}
+
+.header-spectral a {
+    color: white;
+    text-decoration-style: dotted;
+}
+
+.header-spectral a:hover {
+    text-decoration: none;
+}
+
+.btn {
+    margin-top: 30px;
+    padding: 6px;
+    border: 2px solid red;
+    background-color: transparent;
+    border-radius: 5px;
+    display: inline-block;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+    background-color: rgb(237 73 51);
+    width: 14em;
+    font-size: 15px;
+    text-transform: uppercase;
+}
+
+.header-spectral h1 {
+    margin-bottom: 10px;
+    margin-top: 10px;
+    font-size: 25px;
+    margin-top: 15px;
+    margin-bottom: 15px;
+}
+
+.header-spectral p {
+    margin-top: 25px;
+}
+
+.main-section {
+    background-color: rgb(33 178 166);
+}
+
+.main-section-card {
+    text-align: center;
+    padding: 40px;
+}
+
+.main-section-card h1 {
+    margin-bottom: 20px;
+}
+
+.main-section-card p {
+    color: rgb(200 236 233);
+    margin-top: 20px;
+}
+
+.diamond-shape {
+    transform: rotate(45deg);
+    border-radius: 0.25rem;
+    border: solid 2px rgb(29 156 145);
+    height: 4em;
+    width: 4em;
+    margin-bottom: 30px;
+}
+
+.bi-gem,
+.bi-code-slash,
+.bi-heart {
+    font-size: 30px;
+    display: inline-block;
+    transform: rotate(-45deg);
+    position: relative;
+    top: 14px;
+}
+
+.bi-gem {
+    color: rgb(0 255 204);
+}
+
+.bi-code-slash {
+    color: rgb(118 221 255);
+}
+
+.bi-heart {
+    color: rgb(0 240 255);
+}
+
+.icon-shape {
+    display: flex;
+    gap: 70px;
+    justify-content: center;
+    margin-top: 60px;
+}
+
+.image-spectral {
+    width: 100%;
+    /* height:auto; */
+}
+
+.magna-card {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+}
+
+.section-magna h1,
+.main-section-card h1,
+.header-spectral h1,
+.accumsan-card h1 {
+    letter-spacing: 0.2em;
+    font-weight: 800;
+}
+
+.section-magna {
+    background-color: rgb(43 52 61);
+    padding: 40px;
+    /* height:auto; */
+    /*  padding-left:55px; */
+}
+
+.section-magna p {
+    margin-top: 30px;
+}
+
+.icon-card {
+    background-color: rgb(80 83 147);
+}
+
+.accumsan-card {
+    padding: 40px;
+    text-align: center;
+}
+
+.accumsan-card h1 {
+    margin-bottom: 20px;
+}
+
+.accumsan-card p {
+    margin-top: 20px;
+    color: rgb(207 208 226);
+    padding: 15px;
+}
+
+.icons-description {
+    text-align: center;
+    margin-top: 20px;
+    padding: 20px;
+}
+
+.icons-description h3 {
+    margin-top: 25px;
+    letter-spacing: 0.1em;
+    font-weight: 750;
+}
+
+.icons-description p {
+    color: rgb(207 208 226);
+    padding: 15px;
+    margin-bottom: 20px;
+}
+
+.bi-telegram,
+.bi-laptop,
+.bi-headphones,
+.bi-flag {
+    font-size: 30px;
+    color: rgb(0 255 204);
+}
+
+.icons-description .bi-code-slash,
+.icons-description .bi-heart {
+    font-size: 35px;
+    color: rgb(0 255 204);
+    display: inline-block;
+    transform: rotate(2deg);
+}
+
+.btn-card {
+    background-image: -webkit-linear-gradient(top,
```

Comment:
you used same bg-img for footer and header but you  duplicate it  just give them same class use it or else  use selctor class names